client/src/Components/Navbar/Navlink/index.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react/types-6-0'
+import { Navlink, NavlinkProps } from '.'
+
+export default {
+  title: 'Components/Navlink',
+  component: Navlink
+} as Meta
```

Comment:
Using a type assertion (`as`) here should be avoided, as using it means you lose type safety.
You could instead do this:

```ts
const meta: Meta = {
  title: 'Components/Navlink',
  component: Navlink,
  asdasd: 123, // this would be silently ignored in your above code, but it now causes a compile time error
}

export default meta;
```

This is also true of your Navbar stories.

Reply:
This is fine. This is in the storybook docs, so I would like to keep it the same

Reply:
I disagree with following docs blindly, just because they are not using typescript correctly doesn’t mean we can’t